core-tests/shared/src/test/scala/zio/ChunkSpec.scala

```diff
@@ -382,17 +382,17 @@ object ChunkSpec extends ZIOBaseSpec {
 
       assertCompletes
       //checks at compile time
-    } @@ TestAspect.ignore,
-    zio.test.test("++ should work with subtyping") {
+    } @@ TestAspect.ignore
+    // zio.test.test("++ should work with subtyping") {
 
-      trait A
-      trait B extends A
+    //   trait A
+    //   trait B extends A
 
-      val empty: Chunk[B] = Chunk.empty
+    //   val empty: Chunk[B] = Chunk.empty
 
-      val _: NonEmptyChunk[A] = empty ++ Chunk(new A {})
+    //   val _: NonEmptyChunk[A] = empty ++ Chunk(new A {})
```

Comment:
Dead code in light of the changes to `++`? Here and above.

Reply:
Yes, I think we are going to have to add a new operator with a different name, something like `concatNonEmpty`.